BWT/BWT/BWTAlgorithm.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace BWT
+{
+    using System.Text;
+    public class BWTAlgorithm
+    {
+        public static (string, int) FrontBWT(string input)
+        {
+            int lengthOfString = input.Length;
+            string[] rotations = new string[lengthOfString];
+
+            for (int i = 0; i < lengthOfString; ++i)
+            {
+                rotations[i] = input.Substring(i) + input.Substring(0, i);
+            }
+
+            Array.Sort(rotations);
+
+            string transformed = new string(rotations.Select(s => s[lengthOfString - 1]).ToArray());
```

Comment:
Чтобы взять первый элемент с конца строки, начиная с C# 8 можно писать так
```suggestion
            string transformed = new string(rotations.Select(s => s[^1]).ToArray());
```